app/steps/ui/will/newleft/index.js

```diff
@@ -0,0 +1,21 @@
+const ValidationStep = require('app/core/steps/ValidationStep'),
+    json = require('app/resources/en/translation/will/left.json');
+module.exports = class NewWillLeft extends ValidationStep {
+
+    static getUrl() {
+        return '/new-will-left';
+    }
+
+    nextStepUrl(ctx) {
+        return this.next(ctx).constructor.getUrl('noWill');
+    }
+
+    nextStepOptions() {
+        const nextStepOptions = {
+            options: [
+                {key: 'left', value: json.optionYes, choice: 'withWill'}
+            ]
+        };
+        return nextStepOptions;
+    }
+};
```

Comment:
Please put 'use strict'; on the first line of this file, see https://github.com/hmcts/probate-tech-docs/blob/master/javascript-coding-style.md#strict-mode for an example